### ISSUE TITLE
Heroku Deployment Down

### ISSUE THREAD

**Issue by manishapriya94** [Ally-Guide/amplify-back-end#71]
## TLDR: Heroku is not deploying to ally.guide for frontend 

In Heroku we're seeing the following error:
<img width="1051" alt="Screen Shot 2021-06-17 at 12 35 28 PM" src="https://user-images.githubusercontent.com/9143339/122461696-8e8bb100-cf68-11eb-8d7b-4b3bb7751057.png">

At first in [package.json](https://github.com/Ally-Guide/amplify-front-end/blob/main/package.json) main was not in the correct directory, it now is but even though a new build is refreshed with each install 

<img width="950" alt="Screen Shot 2021-06-17 at 12 36 33 PM" src="https://user-images.githubusercontent.com/9143339/122461796-acf1ac80-cf68-11eb-8a16-67b4f313c6fa.png">

Let @manishapriya know if you have access issues with Heroku

**Comment by manishapriya94** [comment]
Closing this as its been resolved:
@mscoutermarsh: Needed to add some special build pack for Heroku + a static.json file
